### PR TITLE
gnome-vfsmm, libgnomecanvasmm, libgnomeuimm: need C++11

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Fix build, fix alignments.

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
```
  /opt/local/include/glibmm-2.4/glibmm/ustring.h:265:13: error: expected ';' at end of declaration list
    ~ustring() noexcept;
              ^
              ;
```

Vaguely recall we saw something like this before, but cannot remember what was the fix.

**Comment by pmetzger** [comment]
CI failed?

**Comment by barracuda156** [comment]
@pmetzger I do not think we are breaking anything in this PR, likely the ports are broken by Apple clang changes in new systems.

Need to find a fix for that now.

**Comment by pmetzger** [comment]
Be that as it may, you made loads of changes to the portfiles, not just setting the C++ standard, and I can't understand the impact of all of them. Generally it's nicest to separate these things.

File: gnome/gnome-vfsmm/Portfile [hunk 1]
```diff
@@ -1,29 +1,38 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem       1.0
+PortSystem      1.0
```
Comment (ryandesign) [inline gnome/gnome-vfsmm/Portfile:3]: This commit contains both functional changes and whitespace changes to the remainder of the file; this is difficult to review or analyze. Please move substantial whitespace-only changes into a separate commit.

Comment (pmetzger) [reply gnome/gnome-vfsmm/Portfile:3]: I made a similar comment above.

Comment (barracuda156) [reply gnome/gnome-vfsmm/Portfile:3]: Ok, will do

Comment (barracuda156) [reply gnome/gnome-vfsmm/Portfile:3]: @pmetzger @ryandesign Separated for each port.

**Comment by pmetzger** [comment]
Much easier to read. We still need it to pass the CI.

**Comment by pmetzger** [comment]
Poke.

**Comment by barracuda156** [comment]
> Poke.

@pmetzger This one ready.

**Pull request merged**